### PR TITLE
Theme Builder - Debug - New - Load debug util.

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by matipojo — DISMISSED** [review]

File: core/app/assets/js/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import App from './app';
 import { Module as SiteEditor } from '@elementor/site-editor';
 
+import 'elementor-editor/utils/debug';
```
Comment (kobizz) [inline core/app/assets/js/index.js:4]: Export it to elementor-common-modules and use it from there

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/common/assets/js/common.js [hunk 2]
```diff
@@ -35,6 +36,8 @@ class ElementorCommonApp extends elementorModules.ViewModule {
 
 		this.api = new API();
 
+		this.debug = new Debug();
```
Comment (matipojo) [inline core/common/assets/js/common.js:39]: Should be the first thing.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/common/assets/js/common.js (hunk 2 above)
Comment (Nevoss) [reply core/common/assets/js/common.js]: Just did it, and then realize that the debug script depends on the "ajax" module that registered in "initModules" method.

**Review by matipojo — DISMISSED** [review]

File: assets/dev/js/editor/editor-base.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ export default class EditorBase extends Marionette.Application {
 
 	helpers = require( 'elementor-editor-utils/helpers' );
 	imagesManager = require( 'elementor-editor-utils/images-manager' ); // TODO: Unused.
-	debug = require( 'elementor-editor-utils/debug' );
+	debug = elementorCommon.debug;
```
Comment (kobizz) [inline assets/dev/js/editor/editor-base.js:31]: 1. Add deprecation to this property (use getter with deprecation notice)
2. Change all internal cases to use `elementorCommon.debug`

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**